tests/conftest.py

```diff
@@ -5650,6 +5651,30 @@ def patch_consumer_toolbox_with_secret():
     config.switch_ctx(restore_ctx_index)
 
 
+@pytest.fixture(scope="session", autouse=True)
+def toolbox_on_faas_consumer():
+    """
+    Create tools pod on FaaS consumer cluster at the start of a test run while using multicluster configuration
+
+    """
+    from ocs_ci.helpers.managed_services import create_toolbox_on_faas_consumer
```

Comment:
The import should be at the beginning of the file.

Reply:
Added here to avoid circular import